SparkleSpin/Style/Font.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Font.swift
+//  SparkleSpin
+//
+//  Created by Britney Smith on 2/18/19.
+//  Copyright © 2019 Britney Smith. All rights reserved.
+//
+
+import UIKit
+
+struct Font {
+    static let titleFont = UIFont(name: "LIBRARY3AM-Regular", size: 64)
+    static let decorativeFont = UIFont(name: "PoetsenOne-Regular", size: 48)
+    static let subheaderFont = UIFont(name: "saxMono-Regular", size: 40)
+    static let mainBodyFont = UIFont(name: "saxMono-Regular", size: 32)
+    static let wheelBodyFont = UIFont(name: "JUICERegular-Regular", size: 26)
+}
```

Comment:
I really like this folder (`/style`) and all the content there. These files look very well organized and they will come handy once you start using all this. 🙌